config/pmd/ruleset.xml

```diff
@@ -10,13 +10,11 @@
   <exclude-pattern>.*/test/.*</exclude-pattern>
 
   <rule ref="category/java/errorprone.xml">
-    <exclude name="DataflowAnomalyAnalysis"/>
     <exclude name="MissingSerialVersionUID"/>
-    <exclude name="BeanMembersShouldSerialize"/>
   </rule>
 
   <rule ref="category/java/multithreading.xml">
-    <!-- Because it complains for the Map interface in the Resource class.-->
+    <!-- Because it complains about the Map interface in the Resource class.-->
```

Comment:
We share Pub-API's PMD spec, but this isn't relevant for us…weird. Probably need to decide what to do about "Standard" and local colour in another meeting.